src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -5822,14 +5829,6 @@ internal override IOperation VisitPointerIndirectionReference(IPointerIndirectio
             return new PointerIndirectionReferenceExpression(Visit(operation.Pointer), semanticModel: null, operation.Syntax, operation.Type, operation.ConstantValue, IsImplicit(operation));
         }
 
-        public override IOperation VisitDynamicObjectCreation(IDynamicObjectCreationOperation operation, int? captureIdForResult)
-        {
-            return new DynamicObjectCreationExpression(VisitArray(operation.Arguments), ((HasDynamicArgumentsExpression)operation).ArgumentNames,
```

Comment:
>VisitArray [](start = 55, length = 10)

Is this the last usage of this method, the VisitInvalid PR has been merged. Consider removing